attestation-service/verifier/src/se/seattest.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) Copyright IBM Corp. 2024
```

Comment:
Do we need this file? IMO things in `se/mod.rs` are enough.

Reply:
Agreed, I'll drop this file later when the IBM SE crate ready.